### ISSUE TITLE
Missing data in `/json/codenames/*.json` due to case sensitivity

### ISSUE THREAD

**Issue by maliayas** [jaredrummler/AndroidDeviceNames#2]
As an example:

`/json/devices.json` file has below definitions:

```json
{
  "manufacturer": "Kobo",
  "market_name": "Kobo Arc",
  "codename": "zeus",
  "model": "Arc"
},
{
  "manufacturer": "Sony Ericsson",
  "market_name": "Xperia Play",
  "codename": "Zeus",
  "model": "Zeus"
},
```

Note the different cases of `"zeus"` and `"Zeus"`.

Then in `/json/codenames/Zeus.json` file, there is below content:

```json
[
  {
    "manufacturer": "Kobo",
    "market_name": "Kobo Arc",
    "codename": "zeus",
    "model": "Arc"
  }
]
```

This happens for all the similar situations; such as `"vision"`, `"swift"`, `"SCH-i509"`, `"Pegasus"`,.. etc

**Comment by maliayas** [comment]
Thanks but is this fixed actually? For example, content of the `/json/codenames/Zeus.json` still contains one device instead of 2 devices mentioned above. The problem is, "zeus" overwrites "Zeus" because they are treated as different files.

**Comment by jaredrummler** [comment]
I moved the generated JSON to [`/json/devices`](https://github.com/jaredrummler/AndroidDeviceNames/tree/master/json/devices) so now [zeus.json](https://github.com/jaredrummler/AndroidDeviceNames/blob/master/json/devices/zeus.json) has both devices.

It should get the correct device since it checks both the codename (Build.DEVICE) and model (Build.MODEL). I just released 1.0.5 and it should fix this. Waiting for maven to sync.

For now, please try out the [sample APK](https://github.com/jaredrummler/AndroidDeviceNames/blob/master/example/example.apk).

**Comment by jaredrummler** [comment]
This is fixed in 1.0.5.

**Comment by maliayas** [comment]
:+1: Thanks for the quick replies and also for the great project. JSON files are reliable now.
